packages/validation/src/node.ts

```diff
@@ -1,13 +1,41 @@
+import {
+  AnnotationSuppressor,
+  DisabledFixer,
+  FingerprintSuppressor,
+  MonokleValidator,
+  ResourceParser,
+  SchemaLoader,
+} from './commonExports.js';
+import {RemotePluginLoader} from './pluginLoaders/PluginLoader.js';
+import {requireFromStringCustomPluginLoader} from './pluginLoaders/requireFromStringLoader.node.js';
+
 /**
  * Modify the commonExports.ts file if you want to export something for both node and browser environments.
  */
 export * from './commonExports.js';
 
 export * from './pluginLoaders/index.node.js';
 export * from './utils/loadCustomPlugin.node.js';
-
-export * from './createExtensibleMonokleValidator.node.js';
-export * from './createDefaultMonokleValidator.node.js';
 export * from './config/index.node.js';
 export * from './config/read.node.js';
-export * from './wasmLoader/RemoteWasmLoader.node.js';
+export * from './validators/open-policy-agent/wasmLoader/RemoteWasmLoader.node.js';
+
+export function createDefaultNodeMonokleValidator() {
```

Comment:
Anything preventing us from calling it `createDefaultMonokleValidator` without the `Node` part? I remember we wanted to use the same names for both browser and node.